mutmut/__init__.py

```diff
@@ -537,17 +540,26 @@ def mutate_node(node, context):
                 value=getattr(node, 'value', None),
                 children=getattr(node, 'children', None),
             )
-            assert not callable(new)
-            if new is not None and new != old:
-                if context.should_mutate():
-                    context.number_of_performed_mutations += 1
-                    context.performed_mutation_ids.append(context.mutation_id_of_current_index)
-                    setattr(node, key, new)
-                context.index += 1
-
-            # this is just an optimization to stop early
-            if context.number_of_performed_mutations and context.mutation_id != ALL:
-                return
+            if isinstance(new, list) and not isinstance(old, list):
+                # multiple mutations
+                news = new
+            else:
+                # one mutation
+                news = [new]
```

Comment:
"news" as plural of "new" is a bit cute, but not correct English! new_list would be better.